_episodes/01-connecting.md

```diff
@@ -140,7 +141,7 @@ adopt the following convention:
 - `{{ site.workshop_host_prompt }}` when the command is to be entered on a terminal connected to the remote system
 - `$` when it really doesn't matter which system the terminal is connected to.
 
-> ## Being Certain Which System your Terminal is connected to
+> ## Being Certain Which System your Terminal is Connected to
```

Comment:
For consistency, this should be sentence case, *i.e.*, only capitalize Being.

Reply:
Hi Trevor ... thanks for responding. I saw the other header lines, we have a mixed use of "Title Case" and "Sentence case" for the headings (both regular headings and callout headings). The reason for capitalization was because I was the others in that particular episode was using "Title Case" in the vicinity of that heading line. Which convention is adopted for HPC Carpentry materials? Could you confirm?

I notice that even Software Carpentry materials have inconsistent use of letter case ... For example:

* https://swcarpentry.github.io/shell-novice/01-intro/index.html
* https://swcarpentry.github.io/shell-novice/03-create/index.html

Reply:
I'm not sure The Carpentries has a specific policy on this, but sentence case is less distracting and more conversational. It's also the more grammatically correct, since subheadings are not the titles of works.

Reply:
Let me take in your suggestion on this one. I'll open up a discussion at Carpentry discuss mailing list.

Reply:
@tkphd @wirawan0 here's The Carpentries' policy on title/subtitle casing for our lesson material  http://carpentries.github.io/lesson-example/06-style-guide/index.html#title-casing

Reply:
CLARIFICATION: For this PR, I only want to have PR up till commit #c4de5d92b06fb0 . Additional commits are for other PR.